src/Store.js

```diff
@@ -188,14 +194,9 @@ class Store {
     }
 
     // Remove all event listeners
-    this.events.removeAllListeners('load')
-    this.events.removeAllListeners('load.progress')
-    this.events.removeAllListeners('replicate')
-    this.events.removeAllListeners('replicate.progress')
-    this.events.removeAllListeners('replicated')
-    this.events.removeAllListeners('ready')
-    this.events.removeAllListeners('write')
-    this.events.removeAllListeners('peer')
+    for(var event in this.events._events) {
```

Comment:
Maybe I'm going insane but I can't seem to find where `this.events._events` is defined as an iterable with these values?

Reply:
See https://github.com/Gozala/events/blob/master/events.js#L61